src/vnet/interface.c

```diff
@@ -490,6 +490,21 @@ vnet_sw_interface_set_flags_helper (vnet_main_t * vnm, u32 sw_if_index,
   si->flags &= ~mask;
   si->flags |= flags;
 
+  if (si->flags & VNET_SW_INTERFACE_FLAG_ADMIN_UP)
+    {
+      vnet_interface_main_t *im = &vnm->interface_main;
+      u32 thread_index = vlib_get_thread_index ();
+      vlib_set_simple_counter (&im->sw_if_counters[9],
```

Comment:
where does the magic `9` come from?

Reply:
Same as above, will be fixed with defenitions